viewer/src/datamodels/cad/CadModelUpdateHandler.ts

```diff
@@ -3,16 +3,15 @@
  */
 
 import * as THREE from 'three';
+// eslint-disable-next-line prettier/prettier
```

Comment:
what's wrong with it btw?

Reply:
It just becomes a long line, so I could remove it :)